build.gradle

```diff
@@ -256,8 +256,7 @@ configure(javaCodeCheckedProjects) {
         testImplementation platform('org.spockframework:spock-bom:2.1-groovy-3.0')
         testImplementation 'org.spockframework:spock-core'
         testImplementation 'org.spockframework:spock-junit4'
-        testImplementation("org.mockito:mockito-core:3.8.0")
-        testImplementation("org.mockito:mockito-inline:3.8.0")
+        testImplementation("org.mockito:mockito-core:5.6.0")
```

Comment:
As of the 5.3.0 release. mockito-inline is no longer published.  It's functionality is now part of mockito-core